libs/core/src/lib/file-uploader/file-uploader.component.ts

```diff
@@ -245,4 +255,11 @@ export class FileUploaderComponent implements ControlValueAccessor {
         this.inputRef.nativeElement.value = '';
         this.onChange([]);
     }
+
+    /** @hidden */
+    private _checkExtension(file: File, allowedExtensions?: string[]): boolean {
```

Comment:
In what situation is `allowedExtensions` optional?

Reply:
Im assuming the case where all extensions are allowed

Reply:
It shouldn't be there. Anyway it has been moved to another service